contract/src/execute.rs

```diff
@@ -160,12 +191,13 @@ fn add_market(
 }
 
 fn deposit(
-    deps: DepsMut,
-    env: Env,
+    deps: &mut DepsMut,
+    env: &Env,
     info: MessageInfo,
     id: MarketId,
     outcome: OutcomeId,
     funds: Funds,
+    liquidity: Decimal256,
```

Comment:
I believe `liquidity` shouldn't be more than `Decimal256::from_ratio(1u8, 1u8)` ? If that's true, probably we should add some kind of a check somewhere.

Reply:
Good catch, I'll add a test for this and update the code.